### ISSUE TITLE
Extract flight creator from the producer

### ISSUE THREAD

**Issue by dawrut** [nokia-wroclaw/innovative-project-jackdaw#27]
The producent class should have a single responsibility which is sending messages, not conversion from raw input to POJO record. 

Please extract `createFlight` method to a separate class and use it within the producer. Move corresponding unit tests.

**Comment by KacperBieganek** [comment]
Should i also then extract all the validation methods to lets say `DataValidator` class?

**Comment by dawrut** [comment]
Yeah, please do. It might be a static method in this class as it doesn't carry any state.

Good point!
